doc/api/util.md

```diff
@@ -345,6 +345,31 @@ util.format('%% %s');
 Some input values can have a significant performance overhead that can block the
 event loop. Use this function with care and never in a hot code path.
 
+## `util.formatText(format, text)`
```

Comment:
With a name like this, I expect it to be able to do any type of formatting; or something very generic like `formatText('Count: %n', someNumber)`. If the intent here is only to apply color formatting, perhaps this should be named `colorizeText`? Or if bold is included, `styleText`?

Reply:
So, I changed it from `colorText` to `formatText` as `util.inspect.colors` also provides `italic` and `underline` for example. `styleText` sounds good too!

Reply:
Changed to `styleText`.